Documentation/Documentation.css

```diff
@@ -164,6 +166,8 @@ fieldset{
 .markup{
     background-color: var(--box-shadow);
     font-weight: 500;
+    padding: 0.3rem;
```

Comment:
Is it supposed to be in rems? my suggestion is to use px for anything below 10px as values in decimals seem like experimented values, also for such a low value, responsiveness isn't expected.

Reply:
okay thanks for suggestion